### ISSUE TITLE
Auto-pause 

### ISSUE THREAD

**Issue by etoumey** [jtoumey/tss_estimator#16]
How to identify auto-paused data?

Could massively skew results if you stop after a sprint.

**Comment by jtoumey** [comment]
I have a suggestion: in `classify_heartrate` when we calculate the elapsed time, if it exceeds some value (I have to look at the file and see what the usual max duration is for "smart recording") we just discard the time? this is not fool proof. maybe there is some other thing we can test. like if GPS coordinates don't change? 

bug label is coooool

**Comment by jtoumey** [comment]
If I have time, I'll put a text file on github that has some other files that I know has auto-pause for like 15 or more minutes. That will be interesting to see. 

Also, my files are all recorded with EDGE500 and may be one data point per second. We'll see.

**Comment by etoumey** [comment]
Mine is from EDGE500 too I think. It has the option of "smart recording" as well

**Comment by etoumey** [comment]
on second thought, should you get stress credit for autopausing? Your heart is still beating at a higher rate than normal. I guess my main problem is, I don't think garmin takes HR data if it's paused. So you could get erroneous data...

**Comment by etoumey** [comment]
Ok, but if we do trapezoidal sums, that could approximate your recovery across that interval right?

**Comment by jtoumey** [comment]
Since we are trying to match what we get from a power meter, I say that we don't record anything during autopause. If you autopause, obviously that portion does not contain any Watts and thus no additional stress score. Your heart may be beating higher than normal (which may result in physiological stress), but you are not doing anything bike-specific -- pedalling. 

I agree that trapezoidal sums will infer a heart rate during the autopause. I just think that we should ignore autopause. What do you think about that ?

**Comment by etoumey** [comment]
yeah I think we need to not count auto pause. But still need to figure how to identify it
